README.md

```diff
@@ -820,6 +821,11 @@ per successful charge. | Payments, Rebooted. | Accepting credit card payments fr
 * Dash | https://kapeli.com/dash
 * Zeal | https://zealdocs.org | [@zealdocs](https://twitter.com/zealdocs) | Free | Offline documentation browser
 
+### Mail Providers
+
+* Google suite | https://gsuite.google.com/
+* Zoho Mail | www.zohomail.com
```

Comment:
Please conside adding pricing to both entries.

Reply:
Yeah please add pricing in our correct format and fix the Zoho link